trajan/traj.py

```diff
@@ -63,6 +64,44 @@ def __init__(self, ds, obsdim, timedim):
         self.obsdim = obsdim
         self.timedim = timedim
 
+    def num_timesteps(self):
+        if self.timedim in self.ds.sizes:
+            timedim = self.timedim
+        else:
+            logger.warning(f'self.timedim ({self.timedim}) is not an existing dimension! Using instead "obs".')
+            timedim = 'obs'
+        return self.ds.sizes[timedim]
+
+    def num_trajectories(self):
```

Comment:
Is this one necessary? Why not use sizes directly?

Reply:
Yes, you are right. I will do that. However, we then have to check whether time or obs ?

And btw: I found for one dataset that self.timedim was "time", whereas the dataset only had "obs" as a dimension. Looks like an existing error?

Reply:
Was that the averaged dataset?

Reply:
Yes, I guess it was. But this should be handled now - same problem as discussed below.